types/prettier/index.d.ts

```diff
@@ -569,7 +569,7 @@ export namespace doc {
             | LineSuffix
             | LineSuffixBoundary
             | Trim;
-        type Doc = string | Doc[] | DocCommand;
+        type Doc = string | DocCommand | Array<string | DocCommand>;
```

Comment:
I don't understand why you'd like to change this, the original type definition is just as the same as https://github.com/prettier/prettier/blob/main/commands.md#prettiers-intermediate-representation-doc

Reply:
Is that intended to allow recursion - array of array etc?

Reply:
yes